recipes/bbhash/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "bbhash" %}
+{% set version = "0.5.3" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/bbhash-{{ version }}.tar.gz
+  sha256: 7e84dc10962fe09c666ff2eddad02c08b8dd6f83fb73f7023dbf93d7dea30f1d
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler("c") }}
+    - {{ compiler("cxx") }}
+    - cython >=0.29.21
+    - setuptools >=49
+    - python
+    - numpy
+  host:
+    - cython >=0.29.21
+    - numpy
+    - pip
+    - python
+    - setuptools >=49
+  run:
+    - python
+    - setuptools >=49
```

Comment:
Is `setuptools` needed at run time. Typically this is only the case if the library uses `pkg_resources`

Reply:
I'll remove it and see! thanks!

Reply:
removed in 0f063df, and tests pass - thx